addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -314,6 +314,13 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
                     continue;
                 }
 
+                if (record.type === 'childList' &&
+                    [...record.addedNodes, ...record.removedNodes].every(node => (
+                        node.nodeType === Node.TEXT_NODE && node.textContent === '\ufeff')
+                    )) {
+                    continue;
+                }
```

Comment:
Should have a comment IMO.

Reply:
True.

```js
// Do not mark the editable dirty when simply adding/removing
// link zwnbsp since these are just technical nodes that aren't
// part of the user's editing of the document.
```